src/ripple/app/tx/impl/ApplyContext.cpp

```diff
@@ -69,4 +71,57 @@ ApplyContext::visit (std::function <void (
     view_->visit(base_, func);
 }
 
+template<std::size_t... Is>
+TER
+ApplyContext::checkInvariantsHelper(TER terResult, std::index_sequence<Is...>)
+{
+    if (view_->rules().enabled(featureEnforceInvariants))
+    {
+        auto const& section = app.config().section("check_invariants");
+        if (get<bool> (section, "enabled", true))
+        {
+            auto checkers = getInvariantChecks();
```

Comment:
Suggest flipping this `if` to improve readability and reduce indentation of the "real" body of the function.
```
if (!view_->rules().enabled(featureEnforceInvariants) ||
    !get<bool>(app.config().section("check_invariants"), "enabled", true))
  return terResult;

[...the checks...]
```